db/tag.c

```diff
@@ -3244,21 +3125,14 @@ int stag_ondisk_to_ix_blobs(const struct dbtable *db, int ixnum, const char *inb
     return rc;
 }
 
-int stag_to_stag_buf(const char *table, const char *fromtag, const char *inbuf,
+int stag_to_stag_buf(const struct dbtable *table, const char *fromtag, const char *inbuf,
                      const char *totag, char *outbuf,
                      struct convert_failure *reason)
 {
     return _stag_to_stag_buf_flags_blobs(find_tag_schema(table, fromtag), find_tag_schema(table, totag), inbuf, outbuf,
                                          0, reason, NULL /*inblobs*/, NULL /*outblobs*/, 0 /*maxblobs*/, NULL);
 }
 
-int stag_to_stag_buf_tz(struct schema *fromsch, const char *table, const char *inbuf, const char *totag, char *outbuf,
```

Comment:
Can also remove from tag.h

Reply:
sure, I'll keep that in mind for the follow-up PR